Translations/fr.po

```diff
@@ -10723,14 +10716,21 @@ msgid ""
 "Out of darkness, out of mind,\n"
 "Cast down into the Halls of the Blind."
 msgstr ""
-"Je peux voir ce que tu ne peux voir.\n"
-"Vision laiteuse puis yeux noir.\n"
-"Tu te tourneras ils s'en seront allés,\n"
-"Chuchotant leur chanson cachée.\n"
-"Alors tu verras l'impensable,\n"
-"Ombres et lumière indiscernable.\n"
-"Hors des ténèbres, incrédule,\n"
-"Jeté dans le Hall des Aveugles."
+"Tu ne peux le voir,\n"
+"          mais moi je le puis.\n"
+"Vision laiteuse et\n"
+"          prunelles pourries.\n"
+"Quand tu te retournes,\n"
+"          ils s'en sont allés,\n"
+"Tout en chuchotant\n"
+"          leur chanson cachée.\n"
+"Puis l'impossible\n"
+"          tu vois qui t'entoure,\n"
+"Des ombres qui dansent,\n"
+"          où était le jour.\n"
+"À force de nuit,\n"
+"          sans force tu beugles,\n"
+"Abandonné dans le Hall des Aveugles."
```

Comment:
Interesting text disposition

Reply:
This can often be seen in poetry books as a way to cut a verse so the line does not get too long (for aesthetics reasons mostly), but keep the end of the verse where the rhyme happens at the end of the line.

Reply:
The text was indeed too long at places. As I couldn’t find shorter verses, I chose to cut all verses so that it looked better than automatic wrapping.